VideoAPI/VideoApi.AcceptanceTests/Features/Conference.feature

```diff
@@ -3,91 +3,101 @@ Feature: Conference
+	When I send the request to the endpoint
+	Then the response should have the status OK and success status True
+	And a list containing only todays hearings conference details should be retrieved
 
-  Scenario: Get conference details by hearing id
-    Given I have a conference
-    And I have a get details for a conference request by hearing id with a valid username
-    When I send the request to the endpoint
-    Then the response should have the status OK and success status True
-    And the conference details should be retrieved
+Scenario: Get conference details by hearing id
+	Given I have a conference
+	And I have a get details for a conference request by hearing id with a valid username
+	When I send the request to the endpoint
+	Then the response should have the status OK and success status True
+	And the conference details should be retrieved
 
-  Scenario: Get expired conferences
-    Given I have a conference
-    And I have another conference
-    And I close the last created conference
-    And I have a get expired conferences request
-    When I send the request to the endpoint
-    Then the response should have the status OK and success status True
-    And I have an empty list of expired conferences
+Scenario: Get expired conferences
+	Given I have a conference
+	And I have another conference
+	And I close the last created conference
+	And I have a get expired conferences request
+	When I send the request to the endpoint
+	Then the response should have the status OK and success status True
+	And I have an empty list of expired conferences
 
-  Scenario: Close all conferences
-    Given I have a conference
-    And I have another conference
-    And I have a conference for tomorrow
-    And I close all conferences
-    And I have a get expired conferences request
-    When I send the request to the endpoint
-    Then the response should have the status OK and success status True
-    And a list not containing the closed hearings should be retrieved
+Scenario: Close all conferences
+	Given I have a conference
+	And I have another conference
+	And I have a conference for tomorrow
+	And I close all conferences
+	And I have a get expired conferences request
+	When I send the request to the endpoint
+	Then the response should have the status OK and success status True
+	And a list not containing the closed hearings should be retrieved
 
-  @VIH-5827
-  Scenario: Get the judges in hearings today
-    Given I have a conference
-    And the judge status is in hearing
-    And I have a get judges in hearings today
-    When I send the request to the endpoint
-    Then the response should have the status OK and success status True
-    And the Judges in hearings should be retrieved
+Scenario: Get audiorecording expired closed conferences
+	Given I have a conference with audiorecording
+	And I have another conference with audiorecording
+	And I have a conference for tomorrow with audiorecording
+	And I close all conferences
```

Comment:
Please update here to match the criteria of status >0

Reply:
done